app/views/clientsmodal.html

```diff
@@ -41,7 +41,9 @@ <h4 class="modal-title" id="addClientLabel">
       </div>
       <div class="form-group" ng-class="{ 'has-error' : ngError.cert }">
         <label><span>**</span> Certificate</label>
-        <textarea class="form-control" rows="5" ng-model="client.cert" ng-trim="false"></textarea>
+        <select class="form-control" style="width: auto;" ng-model="client.cert" ng-options="cert._id as cert.commonName for cert in trustedCerts">
+          <option value="">Select Certificate</option>
+        </select>
```

Comment:
Would this select option still be needed for clients now that the cert property has been removed and certificates are assigned in the channel route?